src/com/homework/ChatService.java

```diff
@@ -1,43 +1,88 @@
 package com.homework;
 
-import java.util.Scanner;
-import java.util.Date;
 import java.text.SimpleDateFormat;
+import java.util.*;
 
 import static java.lang.System.exit;
 
+
 public class ChatService {
-    Scanner scan = new Scanner(System.in);
-    public String sms;
-
-
-    public void chooseOption() {
-        while (true) {
-            System.out.println("Press Number for Action\n1. Send message\n2. Change current user\n3. Exit chat");
-            String option = scan.nextLine();
-            if (option.equals("1")) {
-                sendMessage();
-            } else if(option.equals("2")){
-                break;
-            } else if (option.equals("3")) {
-                exitChat();
-            } else {
-                System.out.println("Something went wrong. Please be careful and choose one of the options below.");
-            }
+    public static ArrayList<ChatMember> activeMemberList = new ArrayList<>();
```

Comment:
I believe, that it would be better to rewrite this line as 
private List<ChatMember> activeMemberList = new ArrayList<>();

1. Use private instead of public to have the field encapsulated. If outside classes need read/write access, then provide getter/setter. https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html
2. Since in general it is possible to have several ChatService each with its list of members, then the field should not be static.  https://docs.oracle.com/javase/tutorial/java/javaOO/classvars.html
3. Using List<ChatMember> in declaration instead of <ArrayList> helps decouple the code from a specific implementation of the interface, in this case ArrayList 
https://stackoverflow.com/questions/9852831/polymorphism-why-use-list-list-new-arraylist-instead-of-arraylist-list-n